lib/src/server_verify/slider_captcha_provider.dart

```diff
@@ -45,7 +45,12 @@ class SliderCaptchaClientProvider {
   Future<bool> init(BuildContext context) async {
     puzzleSize = await _getSize(puzzleUnit8List);
     pieceSize = await _getSize(pieceUnit8List);
-    ratio = _getRatio(context);
+    // mobile screen width is below 800
+    if (MediaQuery.of(context).size.width < 800) {
```

Comment:
Mobile devices are constantly being developed and expanded dramatically over time. I haven't found any documents that say that the horizontal size of mobile apps is always less than 800. This is really not confirmed at present and in the future. 
However, I will still approve the commit because this can be true for the case of the web running on mobile browsers or devices like tablets, ipads.
In the future, i will open a discussion on this topic.